src/data.js

```diff
@@ -656,3 +656,19 @@ Provisioner.prototype.json = function() {
 
 // Export Provisioner
 exports.Provisioner = Provisioner;
+
+/**
+ * Entity for tracking worker-types.
+ */
+let WorkerType = Entity.configure({
+  version:            1,
+  partitionKey:       Entity.keys.StringKey('workerType'),
```

Comment:
Per https://bugzilla.mozilla.org/show_bug.cgi?id=1384947#c1 I think that should be PartitionKey=provisionerId,            RowKey=workerType (both StringKey, meaning get that value from the properties and use it as the key).